File: kafka-producer-stationdata-boot-lambda/src/main/java/com/cdugga/opendata/lambda/stationdatalambdajava/functions/StationData.java
package com.cdugga.opendata.lambda.stationdatalambdajava.functions;

import lombok.Data;

@Data
public class StationData {

	private int schemeId;
	
	private String schemeShortName;
	
	private int stationId;
	
	private String name;
	
	private String nameIrish;
	
	private int docksCount;
	
	private int bikesAvailable;
	
	private int docksAvailable;
	
	private int status;
	
	private double lattitude;
	
	private double longitude;
	
	private String dateStatus;
	
}

File: kafka-producer-stationdata-boot-kotlin/src/main/kotlin/com/cdugga/streaming/bikeshare/bikesharekotlin/StationData.kt
package com.cdugga.streaming.bikeshare.bikesharekotlin

data class StationData(val schemeId: Int, 
                        val schemeShortName: String,
                        val stationId: Int,
                        val name: String,
                        val nameIrish: String,
                        val docksCount: Int,
                        val bikesAvailable: Int,
                        val docksAvailable: Int,
                        val status: Int,
                        val lattitude: Double,
                        val longitude: Int,
                        val dateStatus: String)